### DISCUSSION TITLE
Paper

### DISCUSSION THREAD

**Discussion by cadop** [cadop/nodeik#3] [Ideas]
Compare fastest known IK solver with nodeIK at many timesteps. 
Compare accuracy and realistic tolerances expected etc. 

Why is NODEik and interesting approach:

- Value of nodeIK to include dynamics, like friction. 

Write up paper, if it seems great, publish in great journal, otherwise ICCASS. 

Next steps, how to add multiple DOF for simultaneous path planning with collision avoidance.

**Comment by cadop** [comment]
https://ieeexplore.ieee.org/document/7363472

**Comment by psh117** [comment]
https://europe.naverlabs.com/research/publications/learning-reachable-manifold-and-inverse-mapping-for-a-redundant-robot-manipulator/

**Comment by psh117** [comment]
If we include dynamics things, what could be the advantages? for nice trajectories?
If you have some ideas. It would be very helpful for an extension.

**Reply by psh117** [reply]
Maybe a target example? https://ieeexplore.ieee.org/abstract/document/6697190
